website/docs/getting-started/setup/instance-configuration/authentication/github-login.md

```diff
@@ -7,63 +7,54 @@ sidebar_position: 2
+    2. **Authorization Callback URL:** append `/login/oauth2/code/github` to the Homepage URL (Eg: `https://app.appsmith.com/login/oauth2/code/github`).
 
-  ![](/img/as_github_auth_config.png)
+    ![Click to expand](</img/Github_App_Config.png>)
 
-  Once you've entered your provided the Client ID and Client Secret, click the **Save & Restart** button at the bottom of the page. This prompts your Appsmith instance with the new configuration in effect.
-
-  * **Configure via docker.env file**:
-
-  On the machine hosting the Appsmith instance, find your `docker.env` file located in `<project-root>/stacks/configuration/`. Update the values for the following keys:
-
-    ```bash
-    # Example configuration in docker.env file
-    # ********* Github OAUth **********
-    APPSMITH_OAUTH2_GITHUB_CLIENT_ID=YOUR_GITHUB_CLIENT_ID
-    APPSMITH_OAUTH2_GITHUB_CLIENT_SECRET=YOUR_GITHUB_CLIENT_SECRET
-    # ******************************
-    ```
-
-    * Configure the `APPSMITH_OAUTH2_GITHUB_CLIENT_ID` and `APPSMITH_OAUTH2_GITHUB_CLIENT_SECRET` fields with the client ID and client secret generated in the step above
-    * Configure `APPSMITH_SIGNUP_ALLOWED_DOMAINS` with a second-level domain name **`(ex: abc.com)`**, to only allow users with an email address from that domain name to log in **`(ex: john@abc.com)`**
-      * **Note**: you can provide several domain names using a comma-separated list
-    * [Restart the Appsmith instance](../)
-
----
+## Configure Appsmith instance  
 
 :::caution
-The Client ID and Client Secret entered aren't verified by any means. Please test it out manually before disabling the Form login authentication method. If Form login is disabled before testing the GitHub authentication configuration setup, it may lead to the user getting stuck on the login screen if the GitHub configuration is invalid. You’ll have to update the environment variables manually to fix this, as you can't update it via the Admin settings page since you won’t be logged in anymore
+Test the Client ID and Client Secret before disabling Form login on Appsmith. This ensures that users can access Appsmith with their Github credentials. To fix an invalid Github configuration, update the environment variables.
 :::
+There are two ways to configure the self-hosted Appsmith instance - 
+- Admin Settings
+- Environment Variables
+
+**Admin Settings:** you can add the Client ID, Client Secret and allowed domains from [Admin Settings](/getting-started/setup/instance-configuration/). Go to *Admin Settings> Authentication> Github*, fill the required fields and click the **Save & Restart** button at the bottom of the page.
+
+![](/img/as_github_auth_config.png)
+
+**Environment Variables:** Navigate to the `docker.env` file located in `<PROJECT_ROOT>/stacks/configuration/`. Update the values for the following keys:
```

Comment:
check how we did for Google. don't write for only Docker. 
<img width="820" alt="Screenshot 2023-03-14 at 13 34 15" src="https://user-images.githubusercontent.com/32227135/224934984-4529532e-2a09-4045-9ac2-bbe74759d90d.png">

Reply:
We need to tell the users where to find the config file. Just saying update the configuration file feels like half of the information. How do we solve this situation?

Reply:
Kubernetes, AWS,  Docker have different configuration files. I don't know if we will be able to address that here. But you can say something like 

Update the values for the following keys in the instance configuration file, for example, in the `docker.env` file for Docker installation and in the `values.yaml` file for Kubernetes.

Reply:
Hey did you add this? Can't see it in the PR

Reply:
Added it in google oauth. The changes in GitHub oauth file didn't get committed for some reason. Added it now.